<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>514</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Map Editor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #0f1a52
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::ClickFocus</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,0,0,0,0,0">
      <item>
       <widget class="QComboBox" name="terrenos">
        <property name="font">
         <font>
          <bold>true</bold>
          <stylestrategy>PreferDefault</stylestrategy>
         </font>
        </property>
        <property name="toolTipDuration">
         <number>-1</number>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox {
    background-color: #3C5B9B;
    color: black;
    border-radius: 6px;
    padding: 4px 8px;
    font-weight: bold;
    border: none;
}

QComboBox::drop-down {
    border: none;
    background-color: #2A4277;
    width: 20px;
}

QComboBox::down-arrow {
    width: 10px;
    height: 10px;
}

QComboBox QAbstractItemView {
    background-color: #3C5B9B;
    color: black;
    selection-background-color: #5A78B4;
}
</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Obstacles</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_6">
        <property name="sizeConstraint">
         <enum>QLayout::SizeConstraint::SetMinAndMaxSize</enum>
        </property>
        <property name="spacing">
         <number>5</number>
        </property>
        <item row="1" column="0">
         <widget class="QPushButton" name="botonCaja3">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #3C5B9B;  
    color: black;
    border-radius: 8px;
    padding: 6px 12px;
    font-weight: bold;
    font-size: 14px;
    border: none;
}
QPushButton:hover {
    background-color: #5A78B4; 
}
QPushButton:pressed {
    background-color: #2A4277;  
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/imagenes/bloque3.png</normaloff>:/imagenes/bloque3.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="botonCaja5">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #3C5B9B;  
    color: black;
    border-radius: 8px;
    padding: 6px 12px;
    font-weight: bold;
    font-size: 14px;
    border: none;
}
QPushButton:hover {
    background-color: #5A78B4; 
}
QPushButton:pressed {
    background-color: #2A4277;  
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/imagenes/bloque5.png</normaloff>:/imagenes/bloque5.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="botonCaja2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #3C5B9B;  
    color: black;
    border-radius: 8px;
    padding: 6px 12px;
    font-weight: bold;
    font-size: 14px;
    border: none;
}
QPushButton:hover {
    background-color: #5A78B4; 
}
QPushButton:pressed {
    background-color: #2A4277;  
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/imagenes/bloque2.png</normaloff>:/imagenes/bloque2.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="botonCaja1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #3C5B9B;  
    color: black;
    border-radius: 8px;
    padding: 6px 12px;
    font-weight: bold;
    font-size: 14px;
    border: none;
}
QPushButton:hover {
    background-color: #5A78B4; 
}
QPushButton:pressed {
    background-color: #2A4277;  
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/imagenes/bloque1.png</normaloff>:/imagenes/bloque1.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="botonCaja4">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #3C5B9B;  
    color: black;
    border-radius: 8px;
    padding: 6px 12px;
    font-weight: bold;
    font-size: 14px;
    border: none;
}
QPushButton:hover {
    background-color: #5A78B4; 
}
QPushButton:pressed {
    background-color: #2A4277;  
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/imagenes/bloque4.png</normaloff>:/imagenes/bloque4.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>5</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Weapons</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_7">
        <property name="sizeConstraint">
         <enum>QLayout::SizeConstraint::SetMinAndMaxSize</enum>
        </property>
        <property name="spacing">
         <number>3</number>
        </property>
        <item row="0" column="0">
         <widget class="QPushButton" name="arma1">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #3C5B9B;  
    color: black;
    border-radius: 8px;
    padding: 6px 12px;
    font-weight: bold;
    font-size: 14px;
    border: none;
}
QPushButton:hover {
    background-color: #5A78B4; 
}
QPushButton:pressed {
    background-color: #2A4277;  
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/imagenes/arma1.png</normaloff>:/imagenes/arma1.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="arma3">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #3C5B9B;  
    color: black;
    border-radius: 8px;
    padding: 6px 12px;
    font-weight: bold;
    font-size: 14px;
    border: none;
}
QPushButton:hover {
    background-color: #5A78B4; 
}
QPushButton:pressed {
    background-color: #2A4277;  
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/imagenes/arma3.png</normaloff>:/imagenes/arma3.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="arma2">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #3C5B9B;  
    color: black;
    border-radius: 8px;
    padding: 6px 12px;
    font-weight: bold;
    font-size: 14px;
    border: none;
}
QPushButton:hover {
    background-color: #5A78B4; 
}
QPushButton:pressed {
    background-color: #2A4277;  
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/imagenes/arma2.png</normaloff>:/imagenes/arma2.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>5</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Bomb zone</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QPushButton" name="zona1">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #3C5B9B;  
    color: black;
    border-radius: 8px;
    padding: 6px 12px;
    font-weight: bold;
    font-size: 14px;
    border: none;
}
QPushButton:hover {
    background-color: #5A78B4; 
}
QPushButton:pressed {
    background-color: #2A4277;  
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/imagenes/zona_bomba.png</normaloff>:/imagenes/zona_bomba.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>5</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="btnGuardar">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #3C5B9B;  
    color: black;
    border-radius: 8px;
    padding: 6px 12px;
    font-weight: bold;
    font-size: 14px;
    border: none;
}
QPushButton:hover {
    background-color: #5A78B4; 
}
QPushButton:pressed {
    background-color: #2A4277;  
}
</string>
          </property>
          <property name="text">
           <string>Save map</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnAbrir">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #3C5B9B;  
    color: black;
    border-radius: 8px;
    padding: 6px 12px;
    font-weight: bold;
    font-size: 14px;
    border: none;
}
QPushButton:hover {
    background-color: #5A78B4; 
}
QPushButton:pressed {
    background-color: #2A4277;  
}</string>
          </property>
          <property name="text">
           <string>Open map</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="styleSheet">
       <string notr="true">background-color: #2c3e50</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>884</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
